common/djangoapps/entitlements/models.py

```diff
@@ -172,7 +172,7 @@ class CourseEntitlement(TimeStampedModel):
         blank=True,
         on_delete=models.CASCADE,
     )
-    order_number = models.CharField(max_length=128, null=True)
+    order_number = models.CharField(max_length=128, null=True, blank=True)
```

Comment:
Normal [Django practice](https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.Field.null) suggests avoiding `null=True` for `CharField` to avoid having two different "no data" values. You fix that going forward in `save()`, but an alternative approach would be to also drop `null=True` and add a migration for null->empty. But that's more work.

The only harm in the current approach is that it might be surprising to engineers working with this field to have the "no data" value be null instead of the empty string. But that's fine. And rarely examined explicitly in Python.

Reply:
ok thanks.  Yea I didn't want to do the leg work to determine where if anywhere we are querying this table looking for NULL explicitly (because those queries would need to be changed if now all NULLs are changed to empty string).